mteb/tasks/Audio/Clustering/eng/MusicGenre.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+import librosa
+import numpy as np
+from datasets import Audio
+
+from mteb.abstasks.Audio.AbsTaskAudioClustering import AbsTaskAudioClustering
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class MusicGenreClustering(AbsTaskAudioClustering):
+    label_column_name: str = "label"
+    metadata = TaskMetadata(
+        name="MusicGenreClustering",
+        description="Clustering music recordings in 9 different genres.",
+        reference="https://www-ai.cs.tu-dortmund.de/audio.html",
+        dataset={
+            "path": "mteb/music-genre",
+            "revision": "2ed42a866b5155eb138eb3dc1e68515ccf3c8a50",
+        },
+        type="AudioClustering",
+        category="a2t",
```

Comment:
For clustering we've been treating it as a singular modality task, i.e. i2i or a2a. Let's use `"a2a"` here.